ecommerce/baskets/models.py

```diff
@@ -0,0 +1,33 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from customers.models import Customer
+from products.models import Product
+from core.models import BaseAbstractModel
+from baskets import enums
+
+class Basket(BaseAbstractModel):
+    customer = models.ForeignKey(Customer, verbose_name=_("customer"),
+                                   on_delete=models.PROTECT, null=True)
+    status = models.CharField(choices=enums.Status.choices, verbose_name=_("customer"), max_length=30 )
+    class Meta:
+        verbose_name = _("basket")
+        verbose_name_plural = _("basket")
+    def __str__(self):
+        return f"{self.customer} - {self.Status}"
```

Comment:
I guess self.Status should be **self.status**

Reply:
self.get_status_display()
https://docs.djangoproject.com/en/3.2/ref/models/instances/#extra-instance-methods